commands/cli.go

```diff
@@ -33,6 +33,10 @@ func CreateCli(version string) *cli.App {
 			Usage: "Only delete resources older than this specified value. Can be any valid Go duration, such as 10m or 8h.",
 			Value: "0s",
 		},
+		cli.BoolFlag{
+			Name:  "force",
+			Usage: "Skip nuke confirmation prompt",
```

Comment:
You should put a big "WARNING: this will automatically delete all resources without any confirmation" here :)

Reply:
One other safety measure to consider: When `aws-nuke` is run with `--force`, show the warning but sleep for, say, 5 - 10 seconds so that an interactive user has an opportunity to abort the run.

Reply:
A 10 second "oh shit" pause is a very good idea. https://github.com/gruntwork-io/aws-nuke/issues/14